packages/twilio-run/src/config/start.ts

```diff
@@ -67,12 +67,73 @@ export async function getUrl(cli: StartCliFlags, port: string | number) {
   let url = `http://localhost:${port}`;
   if (typeof cli.ngrok !== 'undefined') {
     debug('Starting ngrok tunnel');
-    const ngrokConfig: NgrokConfig = { addr: port };
+    // Setup default ngrok config, setting the protocol and the port number to
+    // forward to.
+    const defaultConfig: INgrokOptions = { addr: port, proto: 'http' };
+    let tunnelConfig = defaultConfig;
+    let ngrokConfig;
+    if (typeof cli.ngrokConfig === 'string') {
+      // If we set a config path then try to load that config. If the config
+      // fails to load then we'll try to load the default config instead.
+      const configPath = join(process.cwd(), cli.ngrokConfig);
+      try {
+        ngrokConfig = parse(readFileSync(configPath, 'utf-8'));
+      } catch (err) {
+        logger.warn(`Could not find ngrok config file at ${configPath}`);
+      }
+    }
+    if (!ngrokConfig) {
+      // Try to load default config. If there is no default config file, set
+      // `ngrokConfig` to be an empty object.
+      const configPath = join(homedir(), '.ngrok2', 'ngrok.yml');
+      try {
+        ngrokConfig = parse(readFileSync(configPath, 'utf-8'));
```

Comment:
We are already in an async function. Could we turn this into an async file read?

Reply:
Ha, ok. `readFileSync` had already been imported at this point, but happy to use the async version.